Bindings/bindings.json

```diff
@@ -1359,6 +1359,57 @@
                     "help": "Choose platform to send native notifcations. Select Template to send template notification."
                 }
             ]
+        },
+        {
+            "type": "sendGrid",
+            "displayName": "SendGrid",
+            "direction": "out",
+            "settings": [
+                {
+                    "name": "to",
+                    "value": "string",
+                    "defaultValue": "",
+                    "required": false,
+                    "label": "To address",
+                    "help": "To address that the message should be sent to.",
```

Comment:
Nit: The help messages in the portal are formatted in the following manner. May be should change to keep them consistent.
"This is the email address the message should be sent to."